### DISCUSSION TITLE
support transparency

### DISCUSSION THREAD

**Discussion by KiaraGrouwstra** [YaLTeR/niri#231] [Ideas]
now that we can make rules applying to center windows, it would be nice if we were to have transparency support.
one might use this to e.g. set some transparency to terminal windows, as these tend to not need the full opacity as much as graphical applications.

related: #75

**Comment by YaLTeR** [comment]
Makes sense I guess. Although I still need to do more refactoring for rules that apply to visible windows.

**Reply by YaLTeR** [reply]
Implemented in 8b4a9d68e0ba8093e88d20f3a003f78ef27cac0e
